phan.php

```diff
@@ -82,6 +82,10 @@
     ],
     'suppress_issue_types' => [
         'PhanParamSignatureMismatch',
+        'PhanTypeMismatchReturn',
```

Comment:
Let's not add more suppressions. I would rather make the fixes. If you are unsure on how to create stubs - let me know.

Reply:
Removed suppressions